haiku-apps/voxophone/patches/voxophone-1.0.patchset

```diff
@@ -0,0 +1,18 @@
+From fbc0ce8e97430146e9962b6dc10ea91ad8d9e286 Mon Sep 17 00:00:00 2001
+From: Yourself <bach5000@gmail.com>
```

Comment:
You shoud probably replace `Yourself` by `bach5000` and `gmail.com` by `users.noreply.github.com` (which, assembled, produce your github e-mail alias.)

BTW, you might also wish to configure your git user with:
```
git config --global user.name bach5000
git config --global user.email "your github e-mail alias here"
```
then squash your commits, and then
```
git commit --amend --author="bach5000 <your github e-mail alias here>"
```
I've even seen some antispam digits being used in front of the github e-mail alias.

Reply:
Hmmm... I thought I already configured my git things.
I guess I haven't.